cmake/BundledOpenSSL.cmake

```diff
@@ -71,17 +71,37 @@ function(use_openssl SOURCE_DIR BINARY_DIR)
 
     # Note: when upgrading to a later release than 3.1.1 the --no-apps could be used instead of --no-tests to minimize the build size
     if (WIN32)
+        if(MINIFI_USE_JOM_FOR_OPENSSL_BUILD)
+            find_program(JOM_EXECUTABLE_PATH
+                NAMES jom.exe
+                PATHS ENV PATH
+                NO_DEFAULT_PATH)
+            if (NOT JOM_EXECUTABLE_PATH)
+                message(FATAL_ERROR "jom.exe not found. Please install jom and add it to the PATH or turn MINIFI_USE_JOM_FOR_OPENSSL_BUILD option off.")
+            endif()
+            message("Using jom for OpenSSL build: ${JOM_EXECUTABLE_PATH}")
+            include(ProcessorCount)
+            processorcount(jobs)
+            set(OPENSSL_BUILD_COMMAND ${JOM_EXECUTABLE_PATH} -j${jobs})
+            set(OPENSSL_INSTALL_COMMAND ${JOM_EXECUTABLE_PATH} install)
+            set(OPENSSL_WINDOWS_COMPILE_FLAGS /FS)
+        else()
+            message("Using nmake for OpenSSL build")
+            set(OPENSSL_BUILD_COMMAND nmake)
+            set(OPENSSL_INSTALL_COMMAND nmake install)
+            set(OPENSSL_WINDOWS_COMPILE_FLAGS "")
+        endif()
```

Comment:
I would just use jom whenever it's installed, and fall back to nmake when it's not.

Reply:
Updated in 36f6133cbd3c5c280d20c3738a7610d45735e11d